lib/operation.js

```diff
@@ -402,6 +402,27 @@ function flatten (options) {
   return this;
 }
 
+/**
+ * Unflatten - apply transparency (0 /255) to the image using a threshold.
+ *
+ * @example
+ * await sharp(rgbInput)
+ *   .unflatten({ unflattenThresholds: [255] })
```

Comment:
Should this be `threshold` instead of the (internal) `unflattenThresholds`? I think the singular `threshold` might be better for the public API, plus maybe we could allow a `number` as well (then auto-generate a single-element array)?

Reply:
Good call... I have simplified unflatten to convert only white to transparent, and placed it after **threshold** in the pipeline.